src/Service/ServiceExe.cs

```diff
@@ -1816,7 +1817,7 @@ public async Task VerifyContainerConnection(int retrySeconds = 0)
 
                     await ConnectToContainer();
 
-                    if (previousContainerStatus == IsContainerConnected)
+                    if (IsContainerConnected && (previousContainerStatus == IsContainerConnected))

```

Comment:
if (IsContainerConnected && previousContainerStatus) should be equivalent

Reply:
derp you are right